README.rst

```diff
@@ -44,9 +44,9 @@ Dependencies
 ------------
 .. readme_software_requirements
 
-To use PyGranta JobQueue, you must have access to a Granta MI 2024 R2 deployment.
+To use PyGranta JobQueue, you must have access to a Granta MI 2025 R1 deployment.
```

Comment:
We don't really maintain a full cross-compatibility table anywhere, do we? Each version just states the minimum compatible version, and then we include the table that says which PyGranta package version was released for each Granta MI release.

Reply:
True, I've reverted that change.